lib/stitches/response_header.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'allowlist_middleware'
+
+module Stitches
+  class ResponseHeader < Stitches::AllowlistMiddleware
+  protected
+    def do_call(env)
+      status, headers, body = @app.call(env)
+      headers["Content-Type"] = env["CONTENT_TYPE"]
```

Comment:
This was one of our main concerns with this solution. This overwrites the response header for `Content-Type` with a value that we believe is non-standard. For example, what was returned as:
`Content-Type: application/json; charset=utf-8`
would become
`Content-Type: application/json; version=x`, overwriting `charset=utf-8`. According to [RFC 7231 Section 3.1.1.5](https://tools.ietf.org/html/rfc7231#section-3.1.1.5), this appears to be a non-standard value for this header.